mobile/library/swift/EnvoyCxxSwiftInterop/cxx_swift_interop.cc

```diff
@@ -0,0 +1,19 @@
+#import "cxx_swift_interop.h"
+#import "source/server/options_impl.h"
+#import "library/common/engine.h"
+
+namespace Envoy {
+namespace CxxSwift {
+
+void run(BootstrapPtr bootstrap_ptr, std::string log_level, envoy_engine_t engine_handle) {
+  auto options = std::make_unique<Envoy::OptionsImpl>();
+  std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> bootstrap(
+      reinterpret_cast<envoy::config::bootstrap::v3::Bootstrap*>(bootstrap_ptr));
+  options->setConfigProto(std::move(bootstrap));
+  options->setLogLevel(options->parseAndValidateLogLevel(log_level));
```

Comment:
Looks like this takes an `absl::string_view`, so I think run() should probably take that as well.

Reply:
Actually I can construct the `spdlog::level::level_enum` directly on the Swift side, so no string parsing is even needed!